folio-service-tools-spring-dev/src/main/java/org/folio/spring/tools/kafka/KafkaAdminService.java

```diff
@@ -42,6 +46,16 @@ public void createTopics(String tenantId) {
     kafkaAdmin.initialize();
   }
 
+  public void deleteTopicsByTenant(String tenantId) {
+    var configTopics = kafkaProperties.getTopics();
+    var topicsToDelete = configTopics.stream()
+      .map(topic->getTenantTopicName(topic.getName(), tenantId))
+      .filter(name -> name.startsWith(FolioEnvironment.getFolioEnvName() + "." + tenantId + "."))
```

Comment:
There is no need for this line because all strings will startsWith this condition as you use `getTenantTopicName` method